pkg/arvo/lib/hood/drum.hoon

```diff
@@ -212,6 +213,11 @@
     %drum-set-boot-apps  =;(f (f !<(_+<.f vase)) poke-set-boot-apps)
   ==
 ::
+++  on-load
+  |=  %1
+  =<  se-abet  =<  se-view
+  (se-born %home %goad)
+::
```

Comment:
Then we don't know if `lit` was set, so in development ships we'll start a bunch of apps we were trying to avoid.  We maybe should keep that in the state.  I also feel like you should be able to "suspend" an app, and it shouldn't get automatically restarted, though we don't support that yet.

I'm going to leave this for now so we can get the OTA out today, but next time we should consider a more general solution.